Tribler/Test/Core/test_permid.py

```diff
@@ -29,3 +30,25 @@ def test_save_load_keypair_pubkey(self):
         self.assertIsInstance(loaded_key, EC)
 
         shutil.rmtree(unicode(test_base_dir), ignore_errors=True)
+
+    def test_save_load_keypair_pubkey_multichain(self):
+        permid.init()
+        key = permid.generate_keypair_multichain()
+
+        test_base_dir = self.session_base_dir
+        pub_key_path = os.path.join(test_base_dir, 'pub_key_multichain.pem')
+        key_pair_path = os.path.join(test_base_dir, 'pair_multichain.pem')
+
+        permid.save_keypair_multichain(key, key_pair_path)
+        permid.save_pub_key_multichain(key, pub_key_path)
+
+        self.assertTrue(os.path.isfile(pub_key_path))
+        self.assertTrue(os.path.isfile(key_pair_path))
+
+        loaded_key = permid.read_keypair_multichain(key_pair_path)
+        self.assertIsInstance(loaded_key, LibNaCLSK)
+        self.assertEquals(key.key.sk, loaded_key.key.sk)
+        self.assertEquals(key.key.pk, loaded_key.key.pk)
+        self.assertEquals(key.key_to_bin(), loaded_key.key_to_bin())
+
+        shutil.rmtree(unicode(test_base_dir), ignore_errors=True)
```

Comment:
Ah, I based this on the existing test in the file and then semi-fixed it. I will make it properly.